app/controllers/api/v2/assignment_overrides_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V2::AssignmentOverridesController < SharedSecretAuthenticatedApiController
+  include CorsSupport
+
+  def create
+    BatchArbitraryAssignmentCreation.create! create_params.merge(force: true)
+    head :no_content
+  end
+
+  private
+
+  def create_params
+    params.permit(:visitor_id, assignments: %i(split_name variant context))
```

Comment:
`visitor_id` comes from the path, but this works all the same way

right now this is expecting the payload will be:

```
{
  assignments: [...]
}
```

sounds good?